platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -194,6 +267,16 @@ def __init__(self, sfp_index, slot_id=0, linecard_port_count=0, lc_name=None):
             self._thermal_list = initialize_linecard_sfp_thermal(lc_name, slot_id, sfp_index)
 
         self.slot_id = slot_id
+        self.mst_pci_device = self.get_mst_pci_device()
+
+    # get MST PCI device name
+    def get_mst_pci_device(self):
+        device_name = None
+        try:
+            device_name = subprocess.check_output("ls /dev/mst/ | grep pciconf", universal_newlines=True, shell=True).strip()
+        except subprocess.CalledProcessError as e:
+            logger.log_error('Failed to init mst PCI device')
```

Comment:
"Failed to **find** mst PCI device"? and maybe appending error msg from the call with be more informative?